### ISSUE TITLE
Cap the max amount for a mint to 100k by default

### ISSUE THREAD

**Issue by KKA11010** [cashubtc/eNuts#156]
**Is your feature request related to a problem? Please describe.**
It is dangerous for the user to let him stack huge amounts of Ecash in any mint.

**Describe the solution you'd like**
Providing a "safety lock" can emphatize the user on the risks related to this app. The user could disable the option.

**Comment by crimsonleaf363** [comment]
I would like an optional warning. Can you give additional information on what you mean by 'safety lock'? What does it mean for the user? Would a 'safety lock' prevent users from creating more invoices?

**Comment by KKA11010** [comment]
> I would like an optional warning. Can you give additional information on what you mean by 'safety lock'? What does it mean for the user? Would a 'safety lock' prevent users from creating more invoices?

Yes it has to be optional but should be enabled by default. So it means that if you want to mint a new token and the mint balance would exceed 100k sats after the tx, the mint operation will not be performed but instead you will see a warning message that reminds you of the risks and where you can disable it if you want to proceed anyway. This warning will only appear if you want to receive ecash by paying an invoice, not if you claim a token from somewhere else.
